MapboxDirections/MBRouteStep.swift

```diff
@@ -513,6 +513,11 @@ public class RouteStep: NSObject, NSSecureCoding {
      */
     public let destinations: String?
     
+    /**
+     Array of interersections
```

Comment:
And mention the relationship between the size of this array and the next step's `exitIndex` property.